priv/index.html

```diff
@@ -5,43 +5,73 @@
 
 <title>BJukebox</title>
 
-<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700,600' rel='stylesheet' type='text/css'>
+<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,600|Fascinate+Inline|Righteous' rel='stylesheet' type='text/css'>
 <link rel="stylesheet" href="static/site.css" type="text/css" media="screen" />
 
 <script type="text/javascript" src="static/site.js"></script>
 </head>
 
 <body>
-	<h1 id="heading">BJukebox</h1>
-	<div id="starttext">Connecting to WebSocket...</div>
-	<div id="lostconn" style="display:none">Lost connection to WebSocket...retrying</div>
+	<header>
+		<h1 id="heading">BJukebox</h1>
+		<div id="starttext">Connecting to WebSocket...</div>
+		<div id="lostconn" style="display:none">Lost connection to WebSocket...retrying</div>
+	</header>
+
 	<div id="nameentry">
-		<label>Enter your name: <input id="input-name" type="text" autocomplete="username"/></label>
-		<button id="b-entername">Go</button>
-		<span id="nameerror" class="errorbox"></span>
-	</div>
-	<div id="interface">
-		<div id="namedisplaydiv">Your name is <span id="namedisplay" class="name"></span>. <button id="b-changename">Change name</button></div>
-		<label>Queue song URL: <input id="input-songurl" type="text" autocomplete="url"/></label>
-		<button id="b-songurl">Submit</button>
-		<span id="songurlerror" class="errorbox">There was an error!</span>
-		<div id="volumecontrols">
-			<button id="b-voldown">Decrease Volume</button>
-			<button id="b-volup">Increase Volume</button>
-			Current volume:&nbsp;
-			<span id="currentvol">?</span>
+		<h1>Welcome to BJukebox!</h1>
+		<label>Enter your name:</label>
+		<input id="input-name" type="text" autocomplete="username"/><br/>
+		<div>
+			<span></span>
```

Comment:
What is this for?

Reply:
I think I was going to do something with flexbox and align the submit button to the right, but I completely forgot about it this morning. Will remove it in the next commit.